plugins/include-media-export/includeMedia.js

```diff
@@ -0,0 +1,82 @@
+var im = (function () {
+  var element = document.body;
+  var autoUpdate = true;
+  var breakpoints = false;
+
+  function setElement(newElement) {
+    element = newElement;
+  }
+
+  function setUpdateMode(updateMode) {
+    if (updateMode == 'manual') {
+      autoUpdate = false;
+      readBreakpoints();
+    }
+  }
+
+  function readBreakpoints() {
+    if (window.getComputedStyle && (window.getComputedStyle(element, '::after').content != '')) {
+      var data = window.getComputedStyle(element, '::after').content.replace(/'/g, '');
+
+      try {
+        breakpoints = JSON.parse(data);
+      } catch(err) {}
+    }
+  }
+
+  function isBreakpointActive(breakpoint) {
+    if (autoUpdate) {
+      readBreakpoints();
+    }
+
+    return breakpoints.hasOwnProperty(breakpoint) &&
```

Comment:
Any reason to go that specific with `breakpoints[breakpoint].hasOwnProperty('active') && breakpoints[breakpoint].active` rather than `breakpoints[breakpoint].active`?